### PR TITLE
feat(bind): option to skip Cargo.toml from consistency checks

### PR DESCRIPTION
**Pull request by meetmangukiya**
Blocked by https://github.com/gakonst/ethers-rs/pull/1301

## Motivation

https://github.com/foundry-rs/foundry/issues/1701

## Solution

Add `--skip-cargo-toml` option.

### CODE REVIEW

**Review by mattsse — COMMENTED** [review]
needs `cargo +nightly fmt`

File: cli/src/cmd/forge/bind.rs [hunk 1]
```diff
@@ -67,6 +67,10 @@ pub struct BindArgs {
     #[clap(long = "single-file", help = "Generate bindings as a single file.")]
     #[serde(skip)]
     single_file: bool,
+
+    #[clap(long = "skip-cargo-toml", help = "Skip Cargo.toml from consistency checks.")]
```
Comment (mattsse) [inline cli/src/cmd/forge/bind.rs:71]:
```suggestion
    #[clap(long = "skip-cargo-toml", help = "Skip Cargo.toml consistency checks.")]
```

**Review by mattsse — APPROVED** [review]

**Comment by mattsse** [comment]
is this good to merge?

**Comment by meetmangukiya** [comment]
@mattsse this is good to merge, last force push was rebase over master

**Pull request merged**